app/assets/javascripts/govuk_publishing_components/components/step-by-step-nav.js

```diff
@@ -76,8 +76,8 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     var showall = document.createElement('div')
     showall.className = 'gem-c-step-nav__controls'
     showall.innerHTML = '<button aria-expanded="false" class="gem-c-step-nav__button gem-c-step-nav__button--controls js-step-controls-button">' +
-      this.$module.downChevronSvg +
-      this.$module.actions.showAllText +
+      '<span class="gem-c-step-nav__button-text gem-c-step-nav__button-text--all js-step-controls-button-text">' + this.$module.actions.showAllText + '</span>' +
```

Comment:
I've done this too for now, embedded the element. When working on this I started to think [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) would be a good use.  There might be a case for a polyfill.

Reply:
I love template literals and think they would be lush, however I don't want to add any extra weight to the user without good reason. This strikes me as a mere code tidiness case which isn't enough sadly.

Reply:
100%